### ISSUE TITLE
parameters for training model

### ISSUE THREAD

**Issue by freelily** [AimeeKing/crnn-tensorflow#10]
would you pls tell me the parameters you used to train the model?

total epoch, initial learning rate,  the decay rate, and how many epoches processed to reduce the lr?

Thank you.
